player/playloop.c

```diff
@@ -301,10 +305,6 @@ static void mp_seek(MPContext *mpctx, struct seek_params seek)
          (opts->hr_seek >= 0 && seek.type == MPSEEK_ABSOLUTE) ||
```

Comment:
Technically, `MPSEEK_CHAPTER` should be here too since it used to be `MPSEEK_ABSOLUTE`

Reply:
In the switch case, I actually re-assign `seek.type` to `MPSEEK_ABSOLUTE`, because I figured they should be treated exactly the same beyond that point.

Reply:
Oops, I totally missed that somehow.